### ISSUE TITLE
doesn't work on ios safari

### ISSUE THREAD

**Issue by jonenst** [coinchoid/coinchoid-web#1]
We get a blank page. Using jsconsole.com, I got the following logs:

```
https://coinchoid.tomobox.fr/js/vendor.js:40

Error: [$injector:modulerr] http://errors.angularjs.org/1.5.8/$injector/modulerr?p0=Coinchoid&p1=%5B%24injector%3Anomod%5D%20http%3A%2F%2Ferrors.angularjs.org%2F1.5.8%2F%24injector%2Fnomod%3Fp0%3DCoinchoid%0Ahttps%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A6%3A417%0Ahttps%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A25%3A100%0Ab%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A24%3A143%0Ahttps%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A24%3A386%0Ahttps%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A39%3A473%0Aq%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A7%3A359%0Ag%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A39%3A320%0Acb%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A43%3A337%0Ac%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A20%3A392%0ABc%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A21%3A180%0Afe%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A20%3A2%0Ahttps%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A317%3A388%0Ab%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A189%3A488%0ASf%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A37%3A131%0Ad%40https%3A%2F%2Fcoinchoid.tomobox.fr%2Fjs%2Fvendor.js%3A37%3A75
```

```
https://coinchoid.tomobox.fr/js/app.js:43

SyntaxError: Unexpected token '>'
```

**Comment by jonenst** [comment]
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions

```
Desktop
Feature           Chrome   Firefox (Gecko)  Edge         IE         Opera  Safari
Basic support     45.0      22.0 (22.0)     (Yes)    No support      32    No support
```

```
Mobile 
Feature         Android    Android Webview  Firefox Mobile (Gecko)    IE Mobile    Opera Mobile  Safari Mobile Chrome for Android
Basic support  No support     45.0           22.0 (22.0)             No support      No support No support         45.0
```

**Comment by jonenst** [comment]
First error url decoded:
Error: [$injector:modulerr] http://errors.angularjs.org/1.5.8/$injector/modulerr?p0=Coinchoid&p1=[$injector:nomod] http://errors.angularjs.org/1.5.8/$injector/nomod?p0=Coinchoid
https://coinchoid.tomobox.fr/js/vendor.js:6:417
https://coinchoid.tomobox.fr/js/vendor.js:25:100
b@https://coinchoid.tomobox.fr/js/vendor.js:24:143
https://coinchoid.tomobox.fr/js/vendor.js:24:386
https://coinchoid.tomobox.fr/js/vendor.js:39:473
q@https://coinchoid.tomobox.fr/js/vendor.js:7:359
g@https://coinchoid.tomobox.fr/js/vendor.js:39:320
cb@https://coinchoid.tomobox.fr/js/vendor.js:43:337
c@https://coinchoid.tomobox.fr/js/vendor.js:20:392
Bc@https://coinchoid.tomobox.fr/js/vendor.js:21:180
fe@https://coinchoid.tomobox.fr/js/vendor.js:20:2
https://coinchoid.tomobox.fr/js/vendor.js:317:388
b@https://coinchoid.tomobox.fr/js/vendor.js:189:488
Sf@https://coinchoid.tomobox.fr/js/vendor.js:37:131
d@https://coinchoid.tomobox.fr/js/vendor.js:37:75

**Comment by jonenst** [comment]
the service worker sw.js is getting copied in app.js. It's not supposed to. How do we prevent that ?

**Comment by jonenst** [comment]
fixed by 6c8a0cff41
